app/actions/QuoteActions.js

```diff
@@ -79,12 +80,14 @@ export function fetchQuote(quoteId: number) {
   });
 }
 
-export function fetchRandomQuote() {
+export function fetchRandomQuote(seenQuotes?: Array<ID> = []) {
+  const queryString = `?seen=[${String(seenQuotes)}]`;
```

Comment:
Why do you want to send seenQuotes from the client? Isn't it easier to store seenQuotes in the DB when the random quote ID is generated in the backend? 😸The query strings can get quite large

Reply:
You sure? This feels like a front-end thing to me. The query string can indeed get quite large, but storing it backend would mean incurring multiple extra database queries for each request. I can't imagine a long query string worse performance-wise than that. 

Further, which quotes you've "seen" seems like a per-session feature, right? I'm not sure it's desired functionality for that sort of memory to persist between sessions. There could be months between those sessions, which means "resetting" makes sense to me. I'm not saying your point is invalid, thank you for the input! If we want long-term persistence then I agree we can store it back-end, it would just be a different design decision.

Reply:
I see your point so we can stick to queries 💯